.gitignore

```diff
@@ -33,3 +33,6 @@
 
 # Project files
 *.geany
+
+subprojects/nlohmann_json
+builddir
```

Comment:
Isn't the default for meson `build/` ?

Reply:
There doesn't exist a default in meson, you always write a name of a folder. I just chose the one they had in the examples in the documentation site.

Reply:
I see. Thanks c: